### ISSUE TITLE
Disable caching

### ISSUE THREAD

**Issue by chrisngobanh** [wbyoung/babel-plugin-transform-postcss#39]
There should be an option to disable caching, and perhaps have it disabled by default. I was really confused about the behavior of the plugin until I realized that there was caching involved.

**Comment by wbyoung** [comment]
PR welcome… :wink:

**Comment by wbyoung** [comment]
Actually… what were you seeing happening? In theory, caching should be transparent… if you haven't changed a file, then we shouldn't recompile.

Currently the cache doesn't account for config file changes or plugin changes. The former, I think we should handle. The later, I think would be covered by the ability to disable the cache (probably via an environment variable).

Other than that, I don't think it should be disabled (so not by default). This is kind of like Babel's cache which is enabled by default, but can be disabled.

**Comment by chrisngobanh** [comment]
Yeah, the confusion came when I updated my config and didn't see any changes. I probably should have known better because it was documented in the README.

I can see where you're coming from and I completely agree now. The changes to caching I hope to see are:

* Changes to the PostCSS config file should invalidate the cache.
* Add an option to enable/disable the caching function (enabled by default).
